src/index.html

```diff
@@ -1,13 +1,424 @@
+            </a>
+
+            <div class="actions">
+              <a
+                href="#"
+                class="icon nav__icon icon--close"
+              >
+              </a>
+            </div>
+          </div>
+
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#home" class="nav__link">
+                Home
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#about-us" class="nav__link">
+                About Us
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#compare-bikes" class="nav__link">
+                Compare Bikes
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#details" class="nav__link">
+                Details
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#contact-us" class="nav__link">
+                Contacts
+              </a>
+            </li>
+          </ul>
+          <a
+            href="tel:+1 234 5555-55-55"
+            class="nav__call nav__call--phone"
+          >
+            +1 234 5555-55-55
+          </a>
+
+          <a
+            href="tel:+1 234 5555-55-55"
+            class="nav__call nav__call--action"
+          >
+            Book a test ride
+          </a>
+        </div>
+      </div>
+    </nav>
+
+    <main>
+      <section
+        class="page__section about"  id="about-us"
+      >
+        <div class="container">
+          <div class="grid grid--tablet grid--desktop">
+            <div class="grid__item grid__item--desktop-1-4 grid__item--tablet-1-3">
+              <h2 class="about__title">
+                Move Free
+              </h2>
+            </div>
+
+            <div class="grid__item grid__item--desktop-7-12 grid__item--tablet-4-6">
+              <p class="about__paragraph">
+                Shift your ride, not gears. Find the fastest way to move in the city as the bike adapts intuitively to power the speed you need.
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section
+        class="page__section compare" id="compare-bikes"
+      >
+        <div class="container">
+          <h2 class="page__section--title">
+            Compare bikes
+          </h2>
+
+          <div class="compare__bikes bikes">
+            <div class="bikes__bike">
+              <img
+                src="./images/photos/Sporty_4.png"
+                alt="Sporty 4"
+                class="bikes__photo"
+              >
+
+              <h3 class="bikes__title">
+                Sporty 4
+              </h3>
+
+              <p class="bikes__description">
+                The iconic frame brought to a new performance height as a sporty, active ride.
+              </p>
+
+              <h3 class="bikes__price">
+                $ 2 590
+              </h3>
+            </div>
```

Comment:
Actually better to make a width limit for this block 392px starting from 744px of the screen width, like in figma:

![6](https://user-images.githubusercontent.com/102480337/218823452-9b8093a3-7a8c-47eb-8636-6cab8b06977b.png)